src/routes/UserSearchContainer.js

```diff
@@ -60,7 +55,7 @@ export function buildQuery(queryParams, pathComponents, resourceData, logger, pr
     2,
   )(queryParams, pathComponents, resourceData, logger, props);
 
-  return mainQuery && `${NOT_SHADOW_USER_CQL} and ${mainQuery}`;
+  return mainQuery && `${mainQuery}`;
```

Comment:
`return mainQuery` or `return makeQueryFunction(...)`

Reply:
The reason to do so is the `makeQueryFunction` will return `null` and the query will fail or return nothing. I may change at most `return mainQuery || '' `

Reply:
That's how it worked before https://github.com/folio-org/ui-users/pull/2543 - https://github.com/folio-org/ui-users/pull/2543/files#diff-71563f03810f6728eb0a46075967cc18b09939d257bd8838b9378d8e769aca98L40

Reply:
I will re-test it and update it, if it works